specification/botservice/resource-manager/Microsoft.BotService/preview/2017-12-01/botservice.json

```diff
@@ -683,7 +683,7 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.BotService/BotServices/listServiceProviders": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.BotService/listAuthServiceProviders": {

```

Comment:
Please remove the other reference to the resource group parameter in this operation:
https://github.com/Azure/azure-rest-api-specs/pull/3280/files#diff-6e7e52c7a9eb6243152b8870ce21ddf4R700

Reply:
This is the error that is currently causing CI to fail.

Reply:
i see...thanks for pointing it out. the error in the ci build seemed puzzling.